src/migrations/20230425094028-resource-title.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => queryInterface.sequelize.transaction(
+    async (transaction) => {
+      const loggedUser = '0';
+      const sessionSig = __filename;
+      const auditDescriptor = 'RUN MIGRATIONS';
+      await queryInterface.sequelize.query(
+        `SELECT
+                set_config('audit.loggedUser', '${loggedUser}', TRUE) as "loggedUser",
+                set_config('audit.transactionId', NULL, TRUE) as "transactionId",
+                set_config('audit.sessionSig', '${sessionSig}', TRUE) as "sessionSig",
+                set_config('audit.auditDescriptor', '${auditDescriptor}', TRUE) as "auditDescriptor";`,
+        { transaction },
+      );
+      await queryInterface.addColumn('Resources', 'title', { type: Sequelize.TEXT, allowNull: true, defaultValue: null }, { transaction });
+    },
```

Comment:
Could we also have a column for the data the title was collected?

Reply:
A good idea can you enter a ticket for this and assign it to me.